<launch>
  <!--action server for moving base-->
  <node name="simple_mover" type="move_action_server" pkg="simple_controller" output="screen">
    <param name="world_frame" value="/odom_combined"/>
    <param name="position_tolerance" value=".05"/>
    <param name="angular_tolerance" value=".05"/>
    <param name="circle_path" value="false"/>
    <param name="max_speed" value="0.5"/>
    <param name="max_angular_speed" value="0.3"/>
  </node>
  <!--action client which calls the server-->
	  
  <node name="mover" pkg="simple_controller" type="test_move_action" output="screen">
    <param name="world_frame" value="/odom_combined"/>
    <param name="radians_per_scan" value=".5"/>
    <param name="path_radius" value="1.0"/>
  </node>
	
</launch>

